cg/store/status_case_filters.py

```diff
@@ -103,22 +103,25 @@ def get_report_supported_data_delivery_cases(cases: Query, **kwargs) -> Query:
 
 def apply_case_filter(
     cases: Query,
-    functions: List[str],
+    functions: List[Callable],
     date: Optional[datetime] = None,
     pipeline: Optional[Pipeline] = None,
 ) -> Query:
     """Apply filtering functions and return filtered results."""
-    filter_map = {
-        "get_cases_for_analysis": get_cases_for_analysis,
-        "get_cases_has_sequence": get_cases_has_sequence,
-        "get_cases_with_loqusdb_supported_pipeline": get_cases_with_loqusdb_supported_pipeline,
-        "get_cases_with_loqusdb_supported_sequencing_method": get_cases_with_loqusdb_supported_sequencing_method,
-        "get_cases_with_pipeline": get_cases_with_pipeline,
-        "get_cases_with_scout_data_delivery": get_cases_with_scout_data_delivery,
-        "get_inactive_analysis_cases": get_inactive_analysis_cases,
-        "get_new_cases": get_new_cases,
-        "get_report_cases_with_valid_data_delivery": get_report_supported_data_delivery_cases,
-    }
     for function in functions:
-        cases: Query = filter_map[function](cases=cases, date=date, pipeline=pipeline)
+        cases: Query = function(cases=cases, date=date, pipeline=pipeline)
     return cases
+
+
+class CaseFilters(Enum):
+    get_cases_has_sequence: Callable = get_cases_has_sequence
+    get_inactive_analysis_cases: Callable = get_inactive_analysis_cases
+    get_new_cases: Callable = get_new_cases
```

Comment:
Wouldn't be better to implement the variable names with capital letters as in [ApplicationFilters](cg/store/status_application_filters.py)

Reply:
done